### ISSUE TITLE
Add a prop to toggle the dropdown list programmatically

### ISSUE THREAD

**Issue by ducwings** [Khan/react-multi-select#16]
Could you add a prop to toggle the dropdown list programmatically?
Something like:

`<MultiSelect showDropdown={this.state.showDropdown} />`

Thanks!

**Comment by JayAshday** [comment]
Agreed that this, or something that provides similar functionality even if it isn't necessarily a prop, would be very useful.
